front/src/styles/scss/applications/operationalStudies/_simulationresults.scss

```
@@ -39,15 +39,24 @@
 
   .osrd-simulation-container {
     position: relative;
-    background-color: white;
     padding: 1rem;
     border-radius: 4px;
 
     &.speedspacechart-container {
+      display: flex;
       background-color: rgb(247, 246, 238);
+      margin-bottom: 28px;
     }
   }
 
+  .simulation-map {
+    position: relative;
+    height: 450px;
+    margin: 0 30px 56px 30px;
+    border-radius: 20px;
+    box-shadow: 0 5px 10px rgba(0, 0, 0, 0.3);
```

If I copy the css for the map in the [mockup](https://www.sketch.com/s/2fb064cd-c483-46ac-bd30-72705fd63e75/p/9559F6FC-69EE-4DB8-BE1A-77F60F10C97F/canvas#Inspect), I get this for the box-shadows : 

```suggestion
    box-shadow: 0px 1px 0px rgba(255, 255, 255, 1) inset, 0px 4px 7px -3px rgba(255, 171, 88, 0.17), 0px 2px 4px rgba(0, 0, 0, 0.22);
```